packages/share/android/src/main/java/io/flutter/plugins/share/MethodCallHandler.java

```diff
@@ -21,23 +21,27 @@
 
   @Override
   public void onMethodCall(MethodCall call, MethodChannel.Result result) {
+    String text;
+    String subject;
```

Comment:
Sorry for not saying this on the last round, but I think this is going to be a bit more readable if we don't declare these up here, since we don't expect to use them again outside of the block. We can just do `String text =` and `String subject =` in the cases that need them.